po/tr.po

```diff
@@ -1069,9 +1070,8 @@ msgstr[0] ""
 msgstr[1] ""
```

Comment:
It looks like this message is left untranslated.
Other than that, looks good.

Reply:
Thanks for noticing! Fixed it in my other PR!

Reply:
Looks good, thanks!